tests/phpunit/integration/tests/Settings.php

```diff
@@ -86,6 +88,7 @@ public function test_on_plugin_uninstall(): void {
 		$this->assertSame( '', get_option( $this->instance::SETTING_NAME_AD_MANAGER_SLOT_ID ) );
 		$this->assertSame( '', get_option( $this->instance::SETTING_NAME_ADSENSE_PUBLISHER_ID ) );
 		$this->assertSame( '', get_option( $this->instance::SETTING_NAME_ADSENSE_SLOT_ID ) );
+		$this->assertSame( null, get_option( $this->instance::SETTING_NAME_MGID_WIDGET_ID ) );
```

Comment:
It would be good to have `assertNull` instead of `assertSame` with `null` value check.

```suggestion
		$this->assertNull( get_option( $this->instance::SETTING_NAME_MGID_WIDGET_ID ) );
```